client/src/component/customer/ReviewPage.js

```diff
@@ -0,0 +1,119 @@
+import { useState, useEffect } from "react";
+
+import { getReviews } from "../../api/googleApi.js";
+
+import { Box, Text } from "@chakra-ui/react";
+
+import { Avatar, Stack } from "@chakra-ui/react";
+import { StarIcon } from "@chakra-ui/icons";
+
+function ReviewPage() {
+  let [apiData, setApiData] = useState(null);
+
+  const getApiData = async () => {
+    setApiData(await getReviews());
```

Comment:
How will you handle API errors here?

Reply:
I will do in next PR. Thank you for pointing out.